tests/test_data.py

```diff
@@ -168,3 +168,24 @@ def test_get_contract_iofo():
     addr, abi = get_contract_info(1234567)
     assert not addr
     assert not abi
+
+
+@pytest.mark.asyncio
+async def test_different_conf_thresholds():
+    """test if a value is dispuable under different confindence thresholds"""
+
+    # ETH/USD
+    query_id = "83a7f3d48786ac2667503a61e8c415438ed2922eb86a2906e4ee66d9a2ce4992"
+    val = 666
+    threshold = 0.05
+
+    # Is disputable
+    disputable = await is_disputable(val, query_id, threshold)
+    assert isinstance(disputable, bool)
+    assert disputable
+
+    threshold = 0.50
+    # Is now not disputable
+    disputable = await is_disputable(val, query_id, threshold)
+    assert isinstance(disputable, bool)
+    assert not disputable
```

Comment:
Does this pass? 666 is like 55% difference from current price of eth (1172).

Reply:
All tests pass :)